src/meta/meta_turtle.c

```diff
@@ -54,6 +54,47 @@ __metadata_init(WT_SESSION_IMPL *session)
 }
 
 /*
+ * __wt_metadata_walk_backup --
+ *	Walk the contents of any hot backup file looking for the given URI.
+ */
+int
+__wt_metadata_walk_backup(
+    WT_SESSION_IMPL *session, const char *uri, bool *in_backup)
+{
+	WT_DECL_ITEM(key);
+	WT_DECL_ITEM(value);
+	WT_DECL_RET;
+	WT_FSTREAM *fs;
+	bool exist;
+
+	/* Look for a hot backup file. We expect to have one. */
+	*in_backup = false;
+	WT_RET(__wt_fs_exist(session, WT_METADATA_BACKUP, &exist));
+	WT_ASSERT(session, exist);
+	WT_RET(__wt_fopen(session, WT_METADATA_BACKUP, 0, WT_STREAM_READ, &fs));
```

Comment:
Since `__wt_fopen` should fail with a non-existent file, is there any reason to have the `__wt_fs_exist` call?